ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor.cs

```diff
@@ -0,0 +1,26 @@
+namespace ExtraDry.Blazor;
+public partial class DryStaticFilter : ComponentBase, IExtraDryComponent
+{
+    /// <inheritdoc cref="IExtraDryComponent.CssClass "/>
+    [Parameter]
+    public string CssClass { get; set; } = string.Empty;
```

Comment:
Like the DryPageQueryView, this does not render a UI and therefore isn't a `IExtraDryComponent`.  These properties are not needed and including them would only serve to confuse a consumer.  Remove the interface and these superfluous properties...

Reply:
Removed  IExtraDryComponent interface & moved code into a single file to mimic DryPageQueryView as I was getting a DRY1500 to implement the interface